libsrc/_DEVELOPMENT/target/zxn/zxn_crt.asm.m4

```diff
@@ -1048,3 +1048,6 @@ ifelse(__STARTUP, 799,
 
    include(`startup/zxn_crt_799.asm.m4')
 ')
+
+INCLUDE "startup/zxn_crt_banked.asm"
```

Comment:
This change adds the contents of zxn_crt_banked for all compilations - increasing their size.

I think to minimise disruption to historic projects I'd prefer that the file is conditionally included based on `CRT_BANKING_SEGMENT` - that is for banked calls to function `CRT_BANKING_SEGMENT` needs to be defined by the user.